src/cljs/chronograph_web/components/form.cljs

```diff
@@ -86,9 +93,9 @@
    (let [value (get @params input-key)]
      (merge {:placeholder (sentence-case input-key)}
             attributes
-            {:on-change #(rf/dispatch [::update form-key input-key (-> %
-                                                                       .-currentTarget
-                                                                       .-value)])
+            {:onChange #(rf/dispatch [::update form-key input-key (-> %
```

Comment:
@adityaathalye @jysandy Why are we doing this?

Reply:
So that it works with the `antd` components. We decided not to do any case conversion for the sake of simplicity.